tag-release-candidate.sh

```diff
@@ -36,29 +12,27 @@ function current_branch() {
   git rev-parse --abbrev-ref @
 }
 
-if ! branch_exists "${release_branch}" ; then
-  git checkout master
-  if [ ! "$(current_branch)" = master ] ; then
-    echo "Must be on master branch"
-    exit 4
-  fi
-  # Ensure local master is up-to-date by pulling its latest version from origin
-  # and fast-forwarding local master:
-  git fetch origin master
-  git merge --ff-only origin/master
-  # Create the release branch:
-  git push origin master:"${release_branch}"
+#if [[ ! "$(git remote get-url origin)" =~ ^git@github.com:weaveworks/eksctl(\.git)?$ ]] ; then
+#  echo "Invalid origin: $(git remote get-url origin)"
+#  exit 3
+#fi
```

Comment:
Can we remove these commented lines if they're not required?

Reply:
Oh, we need these, I commented them out for testing. Will uncomment